CMakeLists.txt

```diff
@@ -63,7 +63,6 @@ if ("${CMAKE_GENERATOR}" MATCHES "Visual Studio")
   else (DEBUG)
```

Comment:
nit: commit title for multi-commit should have mini-descr before colon, and after should say what this particular commit does: neither is there right now.  Should be sthg like `i#1557 cmake location: use output dirs for remaining unix instances".

Reply:
Edited above.